src/main/java/com/spotinst/sdkjava/model/bl/elastigroup/aws/ElastigroupVolume.java

```diff
@@ -0,0 +1,103 @@
+package com.spotinst.sdkjava.model.bl.elastigroup.aws;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Created by Aditya on 11/10/23.
+ */
+
+public class ElastigroupVolume {
```

Comment:
change the classname to ElastigroupVolumes

Reply:
changed